_includes/related_posts.html

```diff
@@ -0,0 +1,46 @@
+{% assign maxRelated = 5 %}
+{% assign minCommonTags =  1 %}
```

Comment:
could you convert these constants (specifically: 5, 1) to parameters specified in `_config.yml`?